src/main/java/seedu/address/model/affiliation/exceptions/AffiliationPersonNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.model.affiliation.exceptions;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+
+/**
+ * Throws a AffiliationPersonNotFoundException exception which is a CommmandException
+ * when the affiliation person is not found in address book.
+ */
+public class AffiliationPersonNotFoundException extends CommandException {
+
+    /**
+     * Constructs a AffiliationPersonNotFoundException.
+     * @param nameOfPersonNotFound The name of person that is not found in address book.
+     */
+    public AffiliationPersonNotFoundException(String nameOfPersonNotFound) {
+        super(nameOfPersonNotFound + " cannot be added as it does not exist in contact");
```

Comment:
Is possible to adjust this to "... cannot be added <ins>as an affiliation</ins> as ..." with full stop?

Reply:
Thanks for pointing it out. Will change it in the next commit.